pkg/v1/sdk/capabilities/discovery/fake.go

```diff
@@ -14,8 +14,9 @@ import (
 // NewFakeClusterQueryClient returns a fake ClusterQueryClient for use in tests.
 func NewFakeClusterQueryClient(resources []*metav1.APIResourceList, scheme *runtime.Scheme, objs []runtime.Object) (*ClusterQueryClient, error) {
 	fakeDynamicClient := dynamicFake.NewSimpleDynamicClient(scheme, objs...)
-	fakeDiscoveryClient := &fakediscovery.FakeDiscovery{Fake: &k8stesting.Fake{
-		Resources: resources,
-	}}
+	fakeDiscoveryClient := &fakediscovery.FakeDiscovery{
```

Comment:
This looks like an unrelated change?

Reply:
Unrelated, but it's easier to read with the change (and more standard). Should I change it back?

Reply:
Okay to keep since you already have this. But in general, if `gofmt` doesn't complain it just adds noise to the review.